packages/markdown-cicero/package.json

```diff
@@ -48,7 +48,7 @@
     "license-check-and-add": "2.3.6"
   },
   "dependencies": {
-    "@accordproject/concerto-core": "3.19.8",
+    "@accordproject/concerto-core": "^3.19.8",
```

Comment:
Why are we moving away from strict version matches? I believe this will install any minor/patch version available >3.19.8 & <4

Reply:
I'm don't think renovate bot / dependabot strictly needs `^` to identify a version change.

I'll defer to @mttrbrts as I don't have previous context on why we had strict version matching.

Reply:
Happy to use strict matching. [chore(*): audit fix](https://github.com/accordproject/markdown-transform/pull/631/commits/152a83052c0c072c24cf1243dcbbe51392cda4cf)

Reply:
Thanks!